moveit_core/constraint_samplers/include/moveit/constraint_samplers/default_constraint_samplers.h

```diff
@@ -69,6 +69,18 @@ class JointConstraintSampler : public ConstraintSampler
   JointConstraintSampler(const planning_scene::PlanningSceneConstPtr& scene, const std::string& group_name)
     : ConstraintSampler(scene, group_name)
   {
+    int rng_seed;
+    if (ros::param::get("~constraint_sampler_random_seed", rng_seed))
```

Comment:
As this parameter is used by the **Joint**ConstraintSampler, use a more specific name?
```suggestion
    if (ros::param::get("~joint_constraint_sampler_random_seed", rng_seed))
```